apps/browser-extension-wallet/.env.defaults

```
@@ -47,7 +47,7 @@ MATOMO_API_ENDPOINT=https://matomo.cw.iog.io/matomo.php
 PUBLIC_POSTHOG_HOST=https://eu.posthog.com
 POSTHOG_EXCLUDED_EVENTS=
 # set this variable to true only in release packages. By having this set to false, we ensure that we will not be using Post Hog production projects tokens in development stage
-PRODUCTION_MODE_TRACKING=false
+PRODUCTION_MODE_TRACKING=true
```

This should stay as false:
```suggestion
PRODUCTION_MODE_TRACKING=false
```